flake8/cython-header.cfg

```diff
@@ -0,0 +1,4 @@
+[flake8]
+filename = *.pxd,*.pxi.in
```

Comment:
Can the template files go in with the normal pyx files? I think they would be closer to that than the header files

Reply:
@xhochy can you check this? I think good to go otherwise

Reply:
`pxd` can go with the normal `pyx` files but the `pxi.in` raise a lot of errors, especially with indentation. I guess that the current indentation is actually sometimes preferred to the flake8 one as it makes the templates more readable.